app-libs/stf/src/getter.rs

```diff
@@ -269,27 +294,92 @@ impl ExecuteGetter for TrustedGetterSigned {
 				Some(attestations.encode())
 			},
 			 */
-			TrustedGetter::ceremonies_aggregated_account_data(
-				who,
-				community_id,
-				_ceremony_index,
-			) => {
-				error!("TrustedGetter ceremonies_aggregated_account_data");
+			TrustedGetter::ceremonies_aggregated_account_data(who, community_id, account_id) => {
+				debug!("TrustedGetter ceremonies_aggregated_account_data");
+				//Todo Master or Me?
+				if !is_ceremony_master(who) {
+					debug!("TrustedGetter ceremonies_aggregated_account_data, return: No master");
+					return None
+				}
 				let aggregated_account_data =
-					EncointerCeremonies::get_aggregated_account_data(community_id, &who);
-				//aggregated_account_data.personal.map(|p| p.encode())
+					EncointerCeremonies::get_aggregated_account_data(community_id, &account_id);
 				Some(aggregated_account_data.encode())
 			},
 			TrustedGetter::ceremonies_assignments(who, community_id, ceremony_index) => {
-				error!("TrustedGetter ceremonies_assignments");
+				debug!("TrustedGetter ceremonies_assignments");
 				// Block getter of confidential data if it is not the CeremonyMaster.
 				if !is_ceremony_master(who) {
-					error!("TrustedGetter ceremonies_assignments, return: No master");
+					debug!("TrustedGetter ceremonies_assignments, return: No master");
 					return None
 				}
 				let assignments = EncointerCeremonies::assignments((community_id, ceremony_index));
 				Some(assignments.encode())
 			},
+			TrustedGetter::ceremonies_meetup_participant_count_vote(
+				who,
+				community_id,
+				ceremony_index,
+				participant_account_id,
+			) => {
+				debug!("TrustedGetter ceremonies_meetup_participant_count_vote");
+				// Todo Master or Me?
+				// Block getter of confidential data if it is not the CeremonyMaster
+				if !is_ceremony_master(who) {
```

Comment:
Why does the ceremony master need to see this? Is this just for development now?

Reply:
Listed in list-attenstees.